cardano_node_tests/tests/test_scripts.py

```diff
@@ -2144,7 +2143,7 @@ def test_spend_reference_script(
             )
             cluster.g_transaction.submit_tx(tx_file=tx_signed, txins=tx_out_spend.txins)
         else:
-            tx_out_spend = cluster.g_transaction.send_tx(
```

Comment:
I would rather leave this one here to make the code more obvious once we decide to implement the TODO below.